start-mongodb.sh

```diff
@@ -1,3 +1,15 @@
 #!/bin/sh
 
-docker run --name mongodb --publish 27017:27017 --detach mongo:$1
+REPL_COMMAND=""
+if [ -n "$INPUT_MONGO_REPL_SET" ]
+then
+    REPL_COMMAND="--replSet $INPUT_MONGO_REPL_SET"
+fi
+
+docker run --name mongodb --publish 27017:27017 --detach mongo:$1 $REPL_COMMAND
+
```

Comment:
I kinda don't like the mixing of named shell variables and numbered args. Maybe we can move towards named variables in general?